src/App.css

```diff
@@ -3,7 +3,15 @@
   This file is deprecated. All styles should be added inline using PandaCSS.
   See: https://panda-css.com/docs/concepts/writing-styles
 */
-
```

Comment:
Can you add these styles to `panda.config.ts`? This file is actually deprecated.